spec/models/event_spec.rb

```diff
@@ -2,27 +2,124 @@
 
 RSpec.describe Event, type: :model do
   let(:user) { User.create(
-      username: "testusername"
-      firstname: "testfirstname"
-      lastname: "testlastname"
-      profilephoto: 1
-      email: 'test@example.com',
-      password: 'password',
-      password_confirmation: 'password'
-    )
-    user.profile_photo.attach(io: File.open('/path/to/profile/photo.jpg'), filename: 'profile_photo.jpg', content_type: 'image/jpeg')
-  }
-end
+    username: "testusername",
+    firstname: "testfirstname",
+    lastname: "testlastname",
+    email: 'test@example.com',
+    password: 'password',
+    password_confirmation: 'password'
+  ) }
+
```

Comment:
Make sure to be consistent with your quotation marks. Since this is Ruby I would recommend single quotes.